tests/unit/test_koji_wrapper.py

```diff
@@ -62,6 +62,20 @@ def test_init_with_koji_wrapper():
     assert test_wrapper.session is my_koji_wrapper.session
 
 
+def test_init_with_koji_wrapper_only():
+    """
+    GIVEN we have a valid KojiWrapper with a session,
+    WHEN a KojiWrapper object is passed as the session to a new KojiWrapper only
+    THEN this object's session is set to the session and url/topurl of the passed object
+    """
+    my_koji_wrapper = build_wrapper(None)
+    test_wrapper = KojiWrapper(session=my_koji_wrapper)
```

Comment:
I think your idea of changing build_wrapper is a good general improvement, it's not needed to test this specific test case as you can see in this line test_wrapper is what is getting a new KojiWrapper that's initialized cleanly from session only.  I'd recommend any refactor to improve the unittests be a separate commit.